cli/util/uploader.go

```diff
@@ -0,0 +1,352 @@
+package util
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/nspcc-dev/neo-go/cli/cmdargs"
+	"github.com/nspcc-dev/neo-go/cli/options"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient"
+	"github.com/nspcc-dev/neo-go/pkg/services/oracle/neofs"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"github.com/nspcc-dev/neofs-sdk-go/client"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/nspcc-dev/neofs-sdk-go/object/slicer"
+	"github.com/nspcc-dev/neofs-sdk-go/session"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	searchBatchSize     = 10000  // Number of objects to search in a batch for finding max block in container.
```

Comment:
Because I meet some cases when search 
```
		prm := client.PrmObjectSearch{}
		filters := object.NewSearchFilters()
		filters.AddFilter(attribute, fmt.Sprintf("%d", 0), object.MatchNumGE)
		filters.AddFilter(attribute, fmt.Sprintf("%d", indexFileSize), object.MatchNumLE)
		prm.SetFilters(filters)

		res, err := neofs.ObjectSearch(ctx.Context, clientSDK, account.PrivateKey(), containerID.String(), prm)
		fmt.Println("Number of objects found: ", len(objectIDs))
		
```
And the result was:
`Number of objects found:  19705`
it was in NeoFS testnet( maybe this is the reason) and full indexFileSize blocks are in the container for sure.

Reply:
We still have to find the reason of this behaviour; but for now we're OK with batched searches.

Reply:
> The result was:

Is there some error returned from SDK along with this result? Maybe connection is closed before the whole set of OIDs is read?